packages/grafana-ui/src/components/Table/TableNG/utils.ts

```diff
@@ -1102,12 +1102,18 @@ export function parseStyleJson(rawValue: unknown): CSSProperties | void {
   }
 }
 
-// Safari 26 introduced rendering bugs which require us to disable several features of the table.
+// Safari 26.0 introduced rendering bugs which require us to disable several features of the table.
+// The bugs were later fixed in Safari 26.2.
 export const IS_SAFARI_26 = (() => {
```

Comment:
Probably want to name this something a bit more generic since it covers multiple versions of 26.x. Perhaps `DISABLE_VIRTUALIZATION_BROWSER` or something more clever. 😂